backend/TreeOfAKind.Infrastructure/Migrations/20201205181528_AddPhotoToTree.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TreeOfAKind.Infrastructure.Migrations
+{
+    public partial class AddPhotoToTree : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Photo",
+                schema: "trees",
+                table: "Trees",
+                type: "nvarchar(255)",
```

Comment:
Might be too small

Reply:
I'm not keeping the names of photos. Just
`azurestorageuri` + `/guid.type`

Reply:
Okay, no need to be stingy here, also it would be beneficient if you would make directories in storage per tree, cause you could find files easier